src/v3/src/util/idxUtils.ts

```diff
@@ -148,6 +148,10 @@ export const buildAuthCoinProps = (
     return { authenticatorKey: IDX_STEP.PIV_IDP };
   }
 
+  if (nextStep?.name === IDX_STEP.REDIRECT_IDVERIFY) {
+    return { authenticatorKey: nextStep?.idp?.id || '' };
```

Comment:
It's ok to return `undefined` from this function if no auth coin should appear

Reply:
Assigning `authenticatorKey` only when idp.id exist. else the existing flow will return undefined.